nodes/nodes-sigstore-using.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="nodes-sigstore-using"]
+= Manage secure signatures with sigstore
+include::_attributes/common-attributes.adoc[]
+:context: nodes-sigstore-using
+
+toc::[]
+
+You can use the sigstore project with {product-title} to improve supply chain security.
+
+// The following include statements pull in the module files that comprise
+// the assembly. Include any combination of concept, procedure, or reference
+// modules required to cover the user story. You can also include other
+// assemblies.
```

Comment:
I'm not going to hold up merge review for this but just a heads up that I think you left some template content in this assembly